platforms/enterprise/enterprise/src/main/java/org/gradle/internal/enterprise/impl/legacy/DevelocityPluginCompatibility.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2025 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.internal.enterprise.impl.legacy;
+
+import org.gradle.util.internal.VersionNumber;
+
+/**
+ * Information about the earliest supported Develocity plugin version.
+ */
+public class DevelocityPluginCompatibility {
+
+    // Gradle versions 9+ are not compatible Gradle Enterprise plugin < 3.13.1
```

Comment:
💅 "not compatible *with*"

I'd also mention here the corresponding plugin ID, to make it clearer